docs/content/en/docs/workflows/debug.md

```diff
@@ -127,27 +183,54 @@ only invokes the real <tt>node</tt> with <tt>--inspect</tt> if running an applic
-wrapper around [`pydevd`](https://github.com/fabioz/PyDev.Debugger) that uses the
-[_debug adapter protocol_ (DAP)](https://microsoft.github.io/debug-adapter-protocol/). 
+Note that a debugging client must first obtain [the inspector UUID](https://github.com/nodejs/node/issues/9185#issuecomment-254872466).  
+
 
-The DAP is supported by Visual Studio Code, [Eclipse LSP4e](https://projects.eclipse.org/projects/technology.lsp4e),
+#### Python (runtime: `python`, protocols: `dap` or `pydevd`)
+
+Python applications are configured to use either  [`debugpy`](https://github.com/microsoft/debugpy/), or
+wrapper around [`pydevd`](https://github.com/fabioz/PyDev.Debugger).  `debugpy` uses the
+[_debug adapter protocol_ (DAP)](https://microsoft.github.io/debug-adapter-protocol/) which 
+is supported by Visual Studio Code, [Eclipse LSP4e](https://projects.eclipse.org/projects/technology.lsp4e),
 [and other editors and IDEs](https://microsoft.github.io/debug-adapter-protocol/implementors/tools/).
-DAP is not yet supported by JetBrains IDEs like PyCharm.
 
-#### .NET Core
+Python application are recognized by:
+- the presence of a standard Python environment variable:
+  `PYTHON_VERSION`, `PYTHONVERBOSE`, `PYTHONINSPECT`, `PYTHONOPTIMIZE`,
+  `PYTHONUSERSITE`, `PYTHONUNBUFFERED`, `PYTHONPATH`, `PYTHONUSERBASE`,
+  `PYTHONWARNINGS`, `PYTHONHOME`, `PYTHONCASEOK`, `PYTHONIOENCODING`,
+  `PYTHONHASHSEED`, `PYTHONDONTWRITEBYTECODE`, or
+- the container command-line invokes `python`, `python2`, or `python3`.
+
+On recognizing a Python-based container image, `debug` rewrites the container image's
+entrypoint to invoke Python using either the `pydevd` or `debugpy` modules:
+```
+python -m debugpy --listen 5678 <app>
+```
+
+or
+```
+python -m pydevd --server --port 5678 <app.py>
+```
+
+{{< alert title="Note" >}}
+As many Python web frameworks use launcher scripts, like `gunicorn`, Skaffold now uses
+a debug launcher that examines the app command-line.
+{{< /alert >}}
+  
+
+#### .NET Core (runtime: `dotnet`, protocols: `vsdbg`)
 
 .NET Core applications are configured to be deployed along with `vsdbg`.
 
-In order to configure your application for debugging, your app must be:
+.NET Core application are recognized by:
+- the presence of a standard .NET environment variable:
+  `ASPNETCORE_URLS`, `DOTNET_RUNNING_IN_CONTAINER`,
+  `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT`, or
+- the container command-line invokes the [dotnet](https://github.com/dotnet/sdk) cli
 
-- Identified as being dotnet-based by having an entrypoint using [dotnet](https://github.com/dotnet/sdk) cli
-  or one of the following environment variables `ASPNETCORE_URLS`, `DOTNET_RUNNING_IN_CONTAINER`,
-  `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT`.
-- Built with the `--configuration Debug` options to disable optimizations.
+Furthermore, your app must be built with the `--configuration Debug` options to disable optimizations.
```

Comment:
is the `Debug` (vs `debug`) uppercase correct here?  Verifying it works as suggested.

Reply:
It worked for me — this contribution is community provided, and it's difficult to test.